### ISSUE TITLE
[Bug]: [People picker][a11y][Voice over][Narrator][Mobile] People picker not announcing suggested contact list

### ISSUE THREAD

**Issue by vdevarapally** [microsoft/fluentui#27892]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Edge
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker

## Actual Behavior
When we type anything in people picker component, Suggested contact list should be announced when voice over or narrator is turned on.

## Expected Behavior
Voice over is only announcing as suggested contacts and not announcing the names in the list.
Reproduction - Open the above URL and turn on voice over and observe the issue

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by vdevarapally** [comment]
Apologies for the delayed response. I was able to reproduce with voice over on safari. Let me attach a recording
